### PR TITLE
Run refresh workflow also for pull requests

### PR DESCRIPTION
**Pull request by ralfhandl**
The workflow wasn't run for https://github.com/SAP/odata-vocabularies/pull/117

### CODE REVIEW

**Review by HeikoTheissen — COMMENTED** [review]
I don't understand why this would help. What does "on: pull_request:" mean?

**Comment by ralfhandl** [comment]
This means that the workflow is triggered every time someone opens or changes a pull request, also from a forked repository, as in #117.

I guess this runs into authorization issues, need to read up more on this topic. Or hope for you picking it up 😄

**Comment by HeikoTheissen** [comment]
I believe this has got nothing to do with the nodejs.yml file. The owner of a forked repository must explicitly activate actions:

> Because this repository contained workflow files when it was forked, we have disabled them from running on this fork. Make sure you understand the configured workflows and their expected usage before enabling Actions on this repository.

**Comment by HeikoTheissen** [comment]
Please check first whether the actions run if @RaizoC enabled them in the forked repository.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: Node CI
 
-on: push
+on:
+  push:
+    branches:
+      - master
+  pull_request_target:
```
Comment (HeikoTheissen) [inline .github/workflows/nodejs.yml:7]: This would then be different from what oasis-tcs/odata-vocabularies uses.

**Comment by ralfhandl** [comment]
Yes, and without this change we are also different.

I added `pull_request_target` in the odata-openapi repository to get checks running on pull requests from forks.

**Comment by HeikoTheissen** [comment]
I forked this repository, enabled actions and committed a change and the action was executed (https://github.com/HeikoTheissen/odata-vocabularies/actions/runs/957020264). So there is no need to change the nodejs.yml, unless you want to have the same settings as in oasis-tcs/odata-vocabularies.

**Pull request closed without merging**